src/main/java/edu/usf/sas/pal/muser/utils/DateTimeUtil.java

```diff
@@ -15,4 +19,18 @@ public static boolean validateDatesProvided(long dateStartMillis, long dateEndMi
         } else return false;
     }
 
+    /**
+     * Gets the time in the string ISO 8601 UTC format from the provided epoch time
+     * @param millis
+     * @return the time in the string ISO 8601 UTC format from the provided epoch time
+     */
+    public static String getDateAndTimeFromMillis(Long millis) {
+        if (millis == null) return "";
+        Date date = new Date(millis);
```

Comment:
A simpler and more efficient way to do this conversion that IIRC should yield the same result is `Instant.ofEpochMillis(millis).toString();` - this avoids instantiating a Date and SimpleDateFormat.

https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#toString--

If for some reason we need to keep SimpleDateFormat we should only instantiate it once as a static variable to avoid having to process the regex on every invocation, for performance reasons.

Reply:
So I tried using the `Instant` class and had the unit tests failing because the time output by using `Instant` also contains fractional seconds from what I understand. This was our expected value: `2019-08-09T20:29:42[]Z` but with `Instant` we get `2019-08-09T20:29:42[.198]Z`. So do you think that we should keep `SimpleDateFormat` instead of `Instant`? I am not sure if we want fractional seconds or not.

Reply:
I'm in favor of keeping fractional seconds, especially if the code is a simpler implementation. This gives us millisecond-level precision if needed in the human-readable date.